python/lsst/daf/butler/cli/butler.py

```diff
@@ -20,13 +20,17 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import click
+from collections import defaultdict
 import logging
 import os
+import yaml
 
 from . import cmd as butlerCommands
 from .utils import to_upper
 from lsst.utils import doImport
 
+localCmdPkg = "lsst.daf.butler.cli.cmd"
```

Comment:
This looking like an import string does lead me to the mistaken impression that you might doImport this later on but the reality is it's a label used to tag the local commands and not something actionable by the code. Maybe add a comment that this is a label describing the local commands.